src/core/dev_api/validation_util.hpp

```diff
@@ -130,6 +126,19 @@ OPENVINO_API void infer_auto_padding(const Shape& image_shape,
                                      CoordinateDiff& padding_above,
                                      CoordinateDiff& padding_below);
 
+OPENVINO_API std::tuple<element::Type, PartialShape, PartialShape> infer_batch_norm_forward(
```

Comment:
I would move it to batch_norm.cpp as it is use there only and should be replaced by proper `shape_infer` functions in shape inference lib. There is JIRA ticket for it.